{"status":"FAIL","msg":"FAIL:\n ReferenceError: _reject is not defined\n    at ../../test262/test/built-ins/Promise/prototype/then/resolve-pending-rejected-non-thenable.js:35\n\n","path":"../../test262/test/built-ins/Promise/prototype/then/resolve-pending-rejected-non-thenable.js"}